docs/tutorials/onnx/inference_on_onnx_model.md

```diff
@@ -104,11 +104,22 @@ We pick a context, GPU if available, otherwise CPU
 ctx = mx.gpu() if mx.test_utils.list_gpus() else mx.cpu()
 ```
 
-And load them into a MXNet Gluon symbol block. For ONNX models the default input name is `input_0`.
+And load them into a MXNet Gluon symbol block. You can obtain the input data names with:
 
 
 ```python
-net = gluon.nn.SymbolBlock(outputs=sym, inputs=mx.sym.var('input_0'))
+data_names = [graph_input for graph_input in sym.list_inputs()
+                      if graph_input not in arg_params and graph_input not in aux_params]
```

Comment:
Wow, that's hard to read :)
Can you add some parenthesis or something?  Or break it up?